whisper/mlx_whisper/cli.py

```diff
@@ -0,0 +1,224 @@
+import argparse
+import os
+import traceback
+import warnings
+
+from .tokenizer import LANGUAGES, TO_LANGUAGE_CODE
+from .transcribe import transcribe
+from .utils import get_writer, optional_float, optional_int, str2bool
+
+
+def build_parser():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        "audio", nargs="+", type=str, help="audio file(s) to transcribe"
+    )
+    parser.add_argument(
+        "--model",
+        default="mlx-community/whisper-tiny",
+        type=str,
+        help="the path to save model files, or the hugging face repo id to use",
+    )
+    parser.add_argument(
+        "--output_dir",
```

Comment:
Most of our CLI tools use `-` instead of `_`. For consistency I would suggest we do the same for the arguments here. Or did you get this from the original Whisper implementation? Maybe it is good to be consistent with that if so?